es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -165,11 +269,45 @@ export default {
         },
         /**
          * Review Comment Truncation Limit
+         * Also indicates whether to show the modal view of component
          */
         commentLimit: {
             default: 225,
             type: [Number, Boolean],
         },
+        /**
+         * Developer Response
+         */
+        response: {
+            required: false,
+            type: [Number, String, Date, null],
+            default: null,
+        },
```

Comment:
Think this should be a string now no?
```suggestion
        response: {
            required: false,
            type: String,
            default: null,
        },
```